_includes/tutorials/rekeying/ksql/code/docker-compose.yml

```diff
@@ -48,7 +51,7 @@ services:
       SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL: 'zookeeper:2181'
 
   ksql-server:
-    image: confluentinc/cp-ksql-server:5.3.0
+    image: colinhicks/ksqldb-cli:5.5.x-latest
```

Comment:
is this just for testing the changes?

Reply:
correct, this will be replaced with a non-test image